server/issue.go

```diff
@@ -43,6 +43,7 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 
 	g.GET("/issue", func(c echo.Context) error {
 		ctx := c.Request().Context()
+		// We do not load the pipeline reduce the size of response payload and complexity of composing the issue list.
```

Comment:
I would rename store.FindIssue to store.FindIssueStripped and put the comment above that method.

Reply:
Good idea.